eZ/Publish/Core/Repository/LocationService.php

```diff
@@ -1000,6 +1009,8 @@ private function checkCreatePermissionOnSubtreeTarget(APILocation $targetParentL
                         [
                             new CriterionSubtree($loadedSubtree->pathString),
                             new CriterionLogicalNot($contentReadCriterion),
+                            // Do not take the same content into consideration as it can have more than one location
+                            new CriterionLogicalNot(new Criterion\ContentId($loadedSubtree->contentId)),
```

Comment:
We don't allow any magic property access anymore due to performance. We need here, instead of `$loadedSubtree->contentId` a strict getter. Due to architecture I think the easiest way would be:
```suggestion
                            new CriterionLogicalNot(
                                new Criterion\ContentId($loadedSubtree->getContentInfo()->getId())
                            ),
```
because it requires a simple addition of 
```php
    public function getId(): int
    {
        return $this->id;
    }
```
to `\eZ\Publish\API\Repository\Values\Content\ContentInfo`
 and a change of the `ContentInfo` PHPDoc:
```php
/**
 * (...)
 * @property-read int $id @deprecated Use {@see ContentInfo::getId} instead
 * (...)
 */
```